### PR TITLE
Update readme content

### PR DESCRIPTION
**Pull request by Racer159**
## Description

This updates the README to reference docs and be visually cleaner.

## Related Issue

None

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## Checklist before merging

- [X] (Optional) Changes have been linted locally with [golangci-lint](https://github.com/golangci/golangci-lint). (NOTE: We haven't turned on lint checks in the pipeline yet so linting may be hard if it shows a lot of lint errors in places that weren't touched by changes. Thus, linting is optional right now.)

### CODE REVIEW

**Comment by Racer159** [comment]
Still need to do copy edits and finalization but this should provide a better structure.

File: README.md [hunk 1]
```diff
@@ -1,306 +1,46 @@
...
+Zarf was initially theorized and demonstrated in research from Naval Postgraduate School which you can read [here](https://calhoun.nps.edu/handle/10945/68688).
 
+## Demo
 
 [![asciicast](https://asciinema.org/a/475530.svg)](https://asciinema.org/a/475530)
 
-## Why is Zarf Needed?
-Most of the software ecosystem assumes your systems have access to the internet. The world (for good reasons) has become more and more dependent upon Software as a Service (SaaS), which assumes a robust connection to the internet and a willingness to inherently trust 3rd party providers. Although this makes sense for most of the world, certain secure systems must operate either fully disconnected, semi-disconnected, or might need the ability to disconnect in case of emergencies (like while under an active cyber attack). Although only a small percentage of systems, these secure systems make up some of the most vital systems globally, such as Aerospace and Defense, Finance, Healthcare, Energy, Water, Sewage, and many Federal, Local, and State Government systems.  
-
-## Explain Zarf Like I'm Ten(ish)
```
Comment (YrrepNoj) [inline README.md:21]: I really like the how much easier it is to consume the README when it's simplified down like this.
Wanted to pose thoughts on two possible additions:

1. Do we think it would be useful to keep an ELI5/10 section? 
    - I think it might be useful for people to get a super quick high-level summary before they make the 'commitment' to click over to the website/docs.
2. Thoughts on adding a `## Getting Started` section?
    - Right now we link to the docs but we con't have a very clear call to action "If you're interested, here's what your next steps could be". Even if it's a short sentence or two that ends up linking to a specific page on the docs website, I think it might be useful information.

Comment (jeff-mccoy) [reply README.md:21]: 1. I want to kill explain it like I'm bleh section here, I think it fits better on the main site or docs personally. 

2. Call to action could be link to "try now" / "QuickStart"

Comment (YrrepNoj) [reply README.md:21]: 👍 

1. Maybe we could at migrate a couple sentences from the ELI5 section to the first couple paragraphs of the README. 


Right now, even though we mention the phrase 'air gap', all we're saying to describe what Zarf does boils down to `Its a static go binary that can pull, package, and install all the things your clusters need to run` and I feel like we're missing some of the 🪄 magic that makes Zarf a tool that really helps deploy things across an air gap.

Comment (jeff-mccoy) [reply README.md:21]: Yeah I was going to play with some rewording on that part in my comments. Also bullet point what you get from zarf would be good as @Madeline-UX brought up in another PR.

Comment (Racer159) [reply README.md:21]: I like the idea of a quick start, and re-titled the "Docs" section "Getting Started" since that is what we want them to do from that section.

File: README.md [hunk 2]
```diff
@@ -1,306 +1,46 @@
-# Zarf - DevSecOps for Air Gap Systems
+# Zarf - DevSecOps for Air Gap
+
+[![Latest Release](https://img.shields.io/github/v/release/defenseunicorns/zarf)](https://github.com/defenseunicorns/zarf/releases)
+[![Zarf Slack Channel](https://img.shields.io/badge/k8s%20slack-zarf-6d87c3)](https://kubernetes.slack.com/archives/C03B6BJAUJ3)
+[![Zarf Documentation](https://img.shields.io/badge/web-zarf.dev-775ba1)](https://zarf.dev/)
```
[the remark this answers is not in the document]
Comment (Racer159) [reply README.md]: Added!

**Comment by jeff-mccoy** [comment]
Love all the badges and how much simpler/cleaner this is overall.  I do want to fiddle with the wording a bit more in the first section.  Feel like we do need a bullet point list of what zarf does / what it's major features are.  Also the supported OS link seems to be dead.  Would like to get a "build passing" badge too, but think we'd need to have an actual k3s test run on master for that to work.  We do it on release right now.  Lastly, we desperately need to update the demo to something more up to date, i can try to do that this week.

**Comment by jeff-mccoy** [comment]
See #632 for some ideas we can look at adding for the bullet point list.

**Comment by Racer159** [comment]
> Love all the badges and how much simpler/cleaner this is overall. I do want to fiddle with the wording a bit more in the first section. Feel like we do need a bullet point list of what zarf does / what it's major features are. Also the supported OS link seems to be dead. Would like to get a "build passing" badge too, but think we'd need to have an actual k3s test run on master for that to work. We do it on release right now. Lastly, we desperately need to update the demo to something more up to date, i can try to do that this week.

Will take a stab at updating that text.  Where do you see the release action output? I only see where we build and push the release packages not the binary itself.

**Comment by YrrepNoj** [comment]
I like the way the [AlexAivazis/survey](https://github.com/AlecAivazis/survey#readme) project does GIFs in their README. It [looks like](https://github.com/AlecAivazis/survey/blob/master/README.md?plain=1#L7) they use gfycat so that playing the GIF doesn't open the link into a new tab and taking you away from the README.

**Review by jeff-mccoy — APPROVED** [review]
This is a huge improvement over the prior readme, thanks!  I made a few changes I'd like you to double-check before we merge @Racer159 or @YrrepNoj
